src/promptflow-azure/promptflow/azure/_storage/blob/client.py

```diff
@@ -63,29 +66,32 @@ def get_datastore_container_client(
 
 def get_default_datastore(
```

Comment:
```suggestion
def _get_default_datastore(
```
Seems like a private method

Reply:
Fixed.